modules/cudacodec/src/cuvid_ffmpeg_api.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef __OPENCV_CUVID_FFMPEG_API_H__
```

Comment:
Please add short license header (here and for other files): https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure
```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```